Monika After Story/game/zz_hangman.rpy

```diff
@@ -439,7 +432,7 @@ label mas_hangman_game_preloop:
 
 # looping location for the hangman game
 label mas_hangman_game_loop:
-    m 1eua "I'll think of a word.{w=0.5}.{w=0.5}.{nw}"
+    m 1eua "I'll think of a word{cps=2}...{/cps}{nw}"
```

Comment:
I'm not sure why you changed this. We use combinations of the `w` and `nw` tags to progress ellipsis, stick to this style, since what you did actually changes how renpy renders this line.

Reply:
Yeah, you're right, I'll revert.